tests/testthat/test-score.R

```diff
@@ -108,7 +108,7 @@ test_that(
       return(y)
     }
 
-    df <- example_binary[model == "EuroCOVIDhub-ensemble" &
+    df <- copy(example_binary)[model == "EuroCOVIDhub-ensemble" &
```

Comment:
```suggestion
    df <- data.table::copy(example_binary)[model == "EuroCOVIDhub-ensemble" &
```

(trying to use `::` in tests because at some point we want to remove the call to `library(data.table)` in the setup script to not accidentally mask any missing functionality.